src/display/api.js

```diff
@@ -1457,7 +1285,9 @@ var WorkerTransport = (function WorkerTransportClosure() {
     this.loadingTask = loadingTask;
     this.pdfDataRangeTransport = pdfDataRangeTransport;
     this.commonObjs = new PDFObjects();
-    this.fontLoader = new FontLoader(loadingTask.docId);
+    this.fontLoader = new FontLoader(loadingTask.docId, {
+      isEvalSupported: getDefaultSetting('isEvalSupported'),
```

Comment:
In https://github.com/mozilla/pdf.js/pull/7126/files#diff-c0a7c347be94f54892dea2ead8b3df4cR39, there's no parameter that takes this `isEvalSupported` argument. And as far as I can tell, `isEvalSupported` is not used within `FontLoader`, so is this line really necessary?